From 29b89676fe8d6c17a7a51e950707e783c2720d3d Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 19 Dec 2024 19:55:58 +0100
Subject: [PATCH 778/954] fix(stm32mp): drivers/stm32_iwdg: remove num_irq

Remove the unused field num_irq in stm32_iwdg_instance struct.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git a78e6fa6347ebe8bbed7f6d915952c9e9c788b7f fix(stm32mp): drivers/stm32_iwdg: remove num_irq]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ifa7ae53d41ac654173b78226e3ed11d812430fa4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/426131
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/iwdg/stm32_iwdg.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/st/iwdg/stm32_iwdg.c b/drivers/st/iwdg/stm32_iwdg.c
index 8ec88beea3..f01660af66 100644
--- a/drivers/st/iwdg/stm32_iwdg.c
+++ b/drivers/st/iwdg/stm32_iwdg.c
@@ -33,7 +33,6 @@
 struct stm32_iwdg_instance {
 	uintptr_t base;
 	unsigned long clock;
-	int num_irq;
 };
 
 static struct stm32_iwdg_instance stm32_iwdg[IWDG_MAX_INSTANCE];
-- 
2.53.0

